src/public/interfaces.ts

```diff
@@ -334,7 +334,8 @@ export interface Context {
   isTeamArchived?: boolean;
 
   /**
-   * The type of the host client. Possible values are : android, ios, web, desktop, rigel
+   * The type of the host client. Possible values are : android, ios, web, desktop, rigel,
```

Comment:
I think it would be good to indicate that rigel is being deprecated.

Reply:
Thanks, added @deprecated annotation.